api-demo/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "scripts": {
     "build": "rimraf ./lib/ && echo -- TYPESCRIPT -- && tsc && echo -- TSLINT -- && tslint --config tslint.json --project . --rules-dir node_modules/tslint-microsoft-contrib",
-    "start": "node ./lib/start.js"
+    "simple": "node ./lib/start.js simple",
+    "advanced": "node ./lib/start.js advanced"
   }
```

Comment:
in the tsdoc package.json, it looks like we aren't running tests

Reply:
Good catch -- fixed

---
In reply to: [224280950](https://github.com/Microsoft/tsdoc/pull/105#discussion_r224280950) [](ancestors = 224280950)